alerts/alerts.yml

```diff
@@ -0,0 +1,53 @@
+groups:
+  - name: Distribution bot. Unexpected exceptions.
+    rules:
+      - alert: DistributionBotUnexpectedErrors
+        expr: increase(distribution_bot_unexpected_exceptions{service="nor-reward-distribution-bot"}[1h]) > 0
+        labels:
+          severity: critical
+          app_team: tooling
+        annotations:
+          summary: "Distribution bot got an unexpected exception."
+          description: "Unexpected exception has happened."
```

Comment:
Let's add here label from metric.

Reply:
by the way, same alert can be added to depositor

Reply:
Done